compiler/codegen/library.cpp

```diff
@@ -454,6 +454,8 @@ static void setupPythonTypeMap() {
   pythonNames[dtReal[FLOAT_SIZE_64]->symbol] = std::make_pair("double", "float");
   pythonNames[dtBool->symbol] = std::make_pair("bint", "bint");
   pythonNames[dtStringC->symbol] = std::make_pair("const char *", "bytes");
+  // TODO: what're the proper map values for c_ptrConst(c_char) to replace c_string?
```

Comment:
Let's slap today's date on this TODO, just for the extra shame factor if it ends up being a while :P